docs/monitoring/spm-faq.md

```diff
@@ -692,14 +683,13 @@ Restart SPM monitor after the above changes.
 
 **My server stopped sending metrics, so why do I still see it under Hosts Filter?**
 
-Filters have 1 day granularity, which means that a server will be
-listed under Hosts filter until 24 hours since it last sent data have
-passed.  For example, if a server stopped sending data at 1 PM and if at
-8 PM you are looking at the last 6 hours of data (for a period from 2 PM
+Filters have 2h day granularity, which means that a server will be
+listed under Hosts filter until 2h hours since it last sent data have
+passed. For example, if a server stopped sending data at 1 PM and if at
+2:30 PM you are looking at the last 1 hour of data (for a period from 2 PM
 until 8 PM) you will not see data from this server on the graph, but you
```

Comment:
Is "for a period from 2 PM until 8 PM" correct?  If I'm looking at last 1 hour of data at 2:30 PM then I'm looking at the period from 1:30 until 2:30, no?

Reply:
Yes. it should be 1:30 PM to 2:30 PM. Fixed